retryable.gemspec

```diff
@@ -1,19 +1,24 @@
-require File.expand_path('../lib/retryable/version', __FILE__)
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'retryable/version'
 
-Gem::Specification.new do |gem|
-  gem.add_development_dependency 'bundler', '~> 1.0'
-  gem.authors = ['Nikita Fedyashev', 'Carlo Zottmann', 'Chu Yeow']
-  gem.description = 'Retryable#retryable, allow for retrying of code blocks.'
-  gem.email = 'nfedyashev@gmail.com'
-  gem.files = %w[CHANGELOG.md LICENSE.md README.md Rakefile retryable.gemspec]
-  gem.files += Dir.glob('lib/**/*.rb')
-  gem.files += Dir.glob('spec/**/*')
-  gem.homepage = 'http://github.com/nfedyashev/retryable'
-  gem.name = 'retryable'
-  gem.license = 'MIT'
-  gem.require_paths = ['lib']
-  gem.required_rubygems_version = '>= 1.3.6'
-  gem.summary = gem.description
-  gem.test_files = Dir.glob('spec/**/*')
-  gem.version = Retryable::Version
+Gem::Specification.new do |spec|
+  spec.name    = 'retryable'
+  spec.version = Retryable::Version
+  spec.authors = ['Nikita Fedyashev', 'Carlo Zottmann', 'Chu Yeow']
+  spec.email   = ['nfedyashev@gmail.com']
+
+  spec.summary     = 'Retrying code blocks in Ruby'
+  spec.description = spec.summary
+  spec.homepage    = 'http://github.com/nfedyashev/retryable'
+  spec.licenses    = ['MIT']
+
+  spec.require_paths = ['lib']
+  spec.files         = Dir['{config,lib,spec}/**/*', '*.md', '*.gemspec', 'Gemfile', 'Rakefile']
+  spec.test_files    = spec.files.grep(%r{^spec/})
+
+  spec.required_ruby_version     = Gem::Requirement.new('>= 1.9')
```

Comment:
@Chubchenko I just checked top 15 of them gems where `retryable` is specified as a dependency. Neither one supports Ruby 1.8 anymore(1.9+ minimum or not specified). So I guess it's about time to do the same.